pkg/manifests/ext_proc.yaml

```diff
@@ -106,6 +108,7 @@ spec:
         request:
           body: Buffered
         response:
+          body: Buffered
```

Comment:
This will disable Streaming. It's OK for local testing, but we should not use Buffered in the repo. 

You can set this to Streamed, and remove the response body parsing logic in HandleResponseBody (which currently assumes Buffered mode only).

Reply:
Discussed offline. Revert this config and add the description in the PR that the latency metrics will be recorded only if the response body mode is Buffered for now.